### PR TITLE
Add an accessor function to PU runtime interface.

### PR DESCRIPTION
**Pull request by varks**

### CODE REVIEW

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "487",
    "commit-sha": "9ba841058b9701504b4bf580e1031c2dabf85186"
  },
  {
    "component": "enforcerd",
    "pr-id": "502",
    "commit-sha": "13398cbca5728ec30026090a41a289f9c26c270d"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "enforcerd",
    "pr-id": "502",
    "commit-sha": "0fd79d35915285d1cccd004a61d4847271e77d3a"
  },
  {
    "component": "trireme-lib",
    "pr-id": "487",
    "commit-sha": "9ba841058b9701504b4bf580e1031c2dabf85186"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "487",
    "commit-sha": "9ba841058b9701504b4bf580e1031c2dabf85186"
  },
  {
    "component": "enforcerd",
    "pr-id": "502",
    "commit-sha": "17727d24239b763d41fd255225a1e5ba4ecda8d5"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "487",
    "commit-sha": "9ba841058b9701504b4bf580e1031c2dabf85186"
  },
  {
    "component": "enforcerd",
    "pr-id": "502",
    "commit-sha": "f97e69463dbf830cf87603be983038ae75f6a32d"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "487",
    "commit-sha": "9ba841058b9701504b4bf580e1031c2dabf85186"
  },
  {
    "component": "enforcerd",
    "pr-id": "502",
    "commit-sha": "adcab068f85e355a3195f44a212b546aa3f23b8a"
  }
]
```

File: policy/runtime.go [hunk 1]
```diff
@@ -242,3 +243,15 @@ func (r *PURuntime) SetServices(services []common.Service) {
...
+func (r *PURuntime) PortMap() map[nat.Port][]string {
+	r.Lock()
+	defer r.Unlock()
+
+	if r.options != nil {
+		return r.options.PortMap
+	}
+
+	return nil
+}
```
Comment (dstiliadis) [inline policy/runtime.go:257]: Please enhance the unit tests for the new methods.

**Review by dstiliadis — DISMISSED** [review]

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "487",
    "commit-sha": "9ba841058b9701504b4bf580e1031c2dabf85186"
  },
  {
    "component": "enforcerd",
    "pr-id": "502",
    "commit-sha": "918e74ae1838ce0f551b18ed9f0f1fa46703125e"
  }
]
```

**Review by amitlimaye — APPROVED** [review]

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "487",
    "commit-sha": "4873f698358d9e4c1b5c15bdd8a23f9174a0628c"
  },
  {
    "component": "enforcerd",
    "pr-id": "502",
    "commit-sha": "022d19190e7b21aed0170360134c39d1477ab06b"
  }
]
```

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "487",
    "commit-sha": "4873f698358d9e4c1b5c15bdd8a23f9174a0628c"
  },
  {
    "component": "enforcerd",
    "pr-id": "502",
    "commit-sha": "022d19190e7b21aed0170360134c39d1477ab06b"
  }
]
```

**Review by amitlimaye — APPROVED** [review]

**Comment by varks** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "component": "trireme-lib",
    "pr-id": "487",
    "commit-sha": "8335bca9b2fe5191f2712662968f53c16aca17f2"
  },
  {
    "component": "enforcerd",
    "pr-id": "502",
    "commit-sha": "022d19190e7b21aed0170360134c39d1477ab06b"
  }
]
```

File: policy/runtime.go (hunk 1 above)
Comment (varks) [reply policy/runtime.go:257]: Done.

**Pull request merged**